etc/di.xml

```diff
@@ -15,6 +15,11 @@
             <argument name="stockManager" xsi:type="object">MageWorx\OrderEditorInventory\Model\Stock\MultiSourceInventoryManager</argument>
         </arguments>
     </type>
+    <type name="MageWorx\OrderEditor\Model\Order\OrderRepository">
```

Comment:
Explain the purpose of this plugin in the comment (`<!-- Support for editing the order pickup location -->`)